### PR TITLE
Step 12 - New Api from The movie DB Added

### PR DESCRIPTION
**Pull request by pablomarinoacn**

### CODE REVIEW

File: app/src/main/java/com/mstudio/superheroarch/remotedatasource/api/RetrofitInstance.kt [hunk 1]
```diff
@@ -5,10 +5,11 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitInstance {
     private const val BASE_URL = "https://rickandmortyapi.com/api/"
+    private const val TMDB_BASE_URL = "https://api.themoviedb.org/3/tv/60625/"
 
-    fun retrofit(): Retrofit {
+    fun retrofit(isMovieDbApi: Boolean = false): Retrofit {
```
Comment (PabloGrelaM) [inline app/src/main/java/com/mstudio/superheroarch/remotedatasource/api/RetrofitInstance.kt:10]: this is not a good idea, neither scalable

File: app/src/main/java/com/mstudio/superheroarch/presentation/detail/CharacterDetailActivity.kt [hunk 2]
```diff
@@ -56,4 +58,17 @@ class CharacterDetailActivity : AppCompatActivity(), CharacterDetailViewTranslat
     override fun showNoInternetConnection() {
         Snackbar.make(findViewById(android.R.id.content), resources.getString(R.string.no_internet_message), Snackbar.LENGTH_LONG).show()
     }
+
+    override fun showEpisodeRatingAndImage(theMovieDbEpisodeRemoteEntity: TheMovieDbEpisodeRemoteEntity) {
```
Comment (PabloGrelaM) [inline app/src/main/java/com/mstudio/superheroarch/presentation/detail/CharacterDetailActivity.kt:62]: you are using ***RemoteEntity on the activity

**Review by PabloGrelaM — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/mstudio/superheroarch/presentation/model/CharacterData.kt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+data class CharacterData(
+    val id: Int,
+    val name: String,
+    val status: String,
+    val image: String,
+    val species: String,
+    val origin: String,
+    val location: String,
+    val episodes: List<String>
```
Comment (PabloGrelaM) [inline app/src/main/java/com/mstudio/superheroarch/presentation/model/CharacterData.kt:13]: do you need the list of episodes?

**Review by PabloGrelaM — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/mstudio/superheroarch/presentation/model/CharacterData.kt (hunk 3 above)
Comment (pablomarinoacn) [reply app/src/main/java/com/mstudio/superheroarch/presentation/model/CharacterData.kt:13]: true

**Review by PabloGrelaM — APPROVED** [review]

**Pull request merged**